spec/lib/config_spec.rb

```diff
@@ -287,11 +287,56 @@
     it "can change the value" do
       Doorkeeper.configure do
         orm DOORKEEPER_ORM
-        client_credentials :from_digest, :from_params
+        client_credentials :from_basic
```

Comment:
`:from_digest` was never a "supported" client authentication method, from what I can tell, so instead I've switched this to an actually supported client authentication method. (I looked through the git history for https://github.com/doorkeeper-gem/doorkeeper/blob/main/lib/doorkeeper/oauth/client/credentials.rb )

Reply:
True. We have some longstanding issue for digest auth

https://github.com/doorkeeper-gem/doorkeeper/issues/984